ambient.d.ts

```diff
@@ -468,10 +468,52 @@ declare module '@mojaloop/central-services-error-handling' {
 }
 declare module '@mojaloop/central-services-stream' {
   import { EventEmitter } from 'events';
-  import { EventActionEnum, EventTypeEnum } from '@mojaloop/central-services-shared';
-  export interface KafkaConsumerConfig {
-    eventType: EventTypeEnum,
-    eventAction: EventActionEnum,
+  export interface GenericMessage<T, U> {
+    value: {
+      from: string,
+      to: string,
+      id: string,
+      content: {
+        uriParams: unknown,
```

Comment:
can't we use something more specific?

Reply:
I just don't know if this will change in the future - I've analysed 3 of the kafka messages where this is a string - but not all of them. I was just thinking if it's unknown, then we won't blindly believe the types here, and be forced to use our brains and double check. Perhaps I'll add a comment along these lines.

Reply:
> I've analysed 3 of the kafka messages
On a side note, can you pass along these example messages so that I get an idea of what a message coming from the central-ledger looks like?

Reply:
https://github.com/vessels-tech/thirdparty-api-adapter/blob/feature/1674-event-server/test_notification_commit.json

Reply:
Thanks you.